source/includes/sharding-warning.rst

```diff
@@ -0,0 +1,11 @@
+
+.. warning::
+   Syncing large document to a sharded destination cluster can cause 
```

Comment:
[suggestion]

Consider breaking this into two statements. Spitballing:

"Syncing large documents to a sharded cluster can initiate chunk migration. Chunk migration may trigger a bug that can result in data loss on that chunk."

Reply:
Fixed